### PR TITLE
Docs: add usages examples to ecctl docs

### PR DESCRIPTION
**Pull request by kilfoyle**
This adds usage examples into the Cloud ecctl guide, basically matching the CRUD examples we have for the public API.  

(This is a duplicate of #277 but I hadn't signed my commits there and creating a new PR seems easier than fixing that.)

### CODE REVIEW

File: docs/ecctl-getting-started.asciidoc [hunk 1]
```diff
@@ -252,47 +252,292 @@ https://golang.org/pkg/text/template/[`text/templates`] on demand.
...
-  "name": "my example cluster",
+  "id": "7229888e7bf8350c7e4d07d7374171c0",
+  "name": "My first ecctl deployment",
   "resources": [
     {
-      "cloud_id": "my_elasticsearch_cluster:REDACTED",
+      "cloud_id": "My_first_ecctl_deployment:dXMtY2VudHJhbDEuZ2NwLmZvdW5kaXQubm8kYjFlZWVjOGQ0YWVlNGY3ZDgxNTM2Zjc1ZjZhN2Y1MDgkM2ViZTAzNmI0NDhkNDc3Y2E2ZTJjZTQ5NmE4ZDQ5ODA=",
       "credentials": {
-        "password": "REDACTED",
+        "password": "qIJuMbL8BFfgoNq5bpjQhDRC",
```
Comment (karencfv) [inline docs/ecctl-getting-started.asciidoc:405]: I realise these are fake credentials, but should we write them out like this? Is it not better to use `REDACTED` ?

File: docs/ecctl-getting-started.asciidoc [hunk 2]
```diff
@@ -252,47 +252,292 @@ https://golang.org/pkg/text/template/[`text/templates`] on demand.
...
+  ]
+}
+--
+
+The response indicates that the request was submitted successfully. It includes the `elastic` user password, which you can use to log in to Kibana or to access the Elasticsearch REST API. Make a note of the deployment ID, which you will use in the next example.
+
+[id="{p}-example-update-deployment"]
+=== Update a deployment
+
+Now that you have used {p} to create a deployment, you can scale it up, by increasing the size of the Elasticsearch data nodes from 4096 to 8192 MB.
```
Comment (karencfv) [inline docs/ecctl-getting-started.asciidoc:437]: The previous JSON payload (create-deployment.json) has size set to 1024 and this one updating to 4096 :)
```suggestion
Now that you have used {p} to create a deployment, you can scale it up, by increasing the size of the Elasticsearch data nodes from 1024 to 4096 MB.
```

File: docs/ecctl-getting-started.asciidoc (hunk 1 above)
Comment (kilfoyle) [reply docs/ecctl-getting-started.asciidoc]: Definitely!  👍 
It's fixed now.

**Comment by kilfoyle** [comment]
@karencfv  Thanks very much for all your help with this!  It'll be nice to have the examples up to date.

I was able to fix my problem about my commits not being signed properly.  The last step would be to merge this PR but I don't seem to have authority for that.  Would you mind merging for me and/or granting me write authority on the repo?

**Comment by karencfv** [comment]
Thank you @kilfoyle! I have merged you PR but also I've given you write permissions for any future contributions. The cloud-docs team only has Nik in it so I added you individually :)

**Pull request merged**